### PR TITLE
KLAR-3040 checkboxTree fix | implemented tree cloning with lodash

### PR DESCRIPTION
**Pull request by ignasveleckisnc**
# What

- What was done in this Pull Request
fixed CheckboxTree component

- How was it before
- Screenshots
- Any other information that may be useful for the dev to review Pull Request

## Testing

- [x] Is this change covered by the unit tests?

- [x] Is this change covered by the integration tests?

- [x] Is this change covered by the automated acceptance tests? (if applicable)

## Compatibility

- [x] Does this change maintain backward compatibility?

### CODE REVIEW

File: src/components/checkboxTree/utils.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { Composition } from "./types";
+import * as _ from "lodash";
```
Comment (matassp) [inline src/components/checkboxTree/utils.ts:2]: `import cloneDeep from "lodash/cloneDeep";`
will only import whats needed

File: package.json [hunk 2]
```diff
@@ -72,9 +72,11 @@
     "styled-components": "^5.0.0"
   },
   "dependencies": {
+    "@types/lodash": "^4.14.178",
```
Comment (matassp) [inline package.json:75]: this should be in `devDependencies`

File: src/components/checkboxTree/utils.ts (hunk 1 above)
Comment (nowyDEV) [reply src/components/checkboxTree/utils.ts:2]: Maybe we could just install a function module from npm?
https://www.npmjs.com/package/lodash.clonedeep
https://www.npmjs.com/package/@types/lodash.clonedeep

File: package-lock.json [hunk 3]
```diff
@@ -9,9 +9,11 @@
       "version": "4.3.0",
       "license": "MIT",
       "dependencies": {
+        "@types/lodash": "^4.14.178",
```
Comment (nowyDEV) [inline package-lock.json:12]: you need to remove -> install it i think, it should be in devDependencies here

**Comment by matassp** [comment]
uncommit the .tgz file

**Review by nowyDEV — APPROVED** [review]

**Review by matassp — APPROVED** [review]

**Pull request merged**